### ISSUE TITLE
Implement async remote files

### ISSUE THREAD

**Issue by Naatan** [Komodo/KomodoEdit#1959]
This has been internally tracked for a while, just opening a bug for it so people publicly can track it also.

**Comment by xTop-SecreTx** [comment]
I've got the same problem. Is there any fix?

**Comment by Naatan** [comment]
This is a bug tracker, the fix will be the resolution of this enhancement.

**Comment by xTop-SecreTx** [comment]
Is the enhancement in progress or finished?

**Comment by Naatan** [comment]
If it is finished this ticket would be closed. Sorry but you'll have to wait, a fix for this is not quiet around the corner. Komodo 11 is still quite a while off.

**Comment by Defman21** [comment]
https://github.com/Komodo/KomodoEdit/commit/03650e4c018e8c34004e1508c51808db4eda839d

**Comment by cgchoffman** [comment]
@Defman21, that patch does NOT fix this.  It only fixes update checking.  This bug is too generic.  There is still a LOT to be done.

**Comment by Defman21** [comment]
Yeah, I understand. I just added it as a reference that something related to the issue got implemented.
